css/ampshared.css

```diff
@@ -518,20 +518,10 @@ amp-accordion > section > * {
 }
 
 amp-accordion,
-amp-accordion > section,
-.i-amphtml-accordion-header,
-.i-amphtml-accordion-content {
+amp-accordion > section {
   margin: 0;
 }
 
-/* heading element*/
-.i-amphtml-accordion-header {
```

Comment:
I'm not sure I 100% understand the context behind this, but I gather at some point this was a selector that worked prior to the extension loading to avoid a FOUC. Also I think it was changed at some point to be a less specific selector (because the more specific one broke some sites)?

Since the accordion is `layout: container` and the header element does not use `box-sizing: border-box;`, the border here will cause content shifting once the extension loads. The padding-right will also cause the text within the header to shift. If this causes the text to wrap onto another line (where it did not before), this will also cause content shifting for subsequent elements on the page.

This PR does not change the state of things, but I'm assuming we are okay with this content shifting post extension load?

Reply:
I'll speak to the move specifically. I checked, and the only place where the `i-amphtml-accordion-*` classes are set is in the extension itself. Thus, having these CSS definitions available before extension is loaded seemed unnecessary.

Reply:
I think the move is fine, its just that that this seemed to avoid a FOUC in the past, but at some point was changed. Moving it out of sight seems to make it less likely that it is fixed in the future (unless this is the behavior we want, for whatever reason).

Reply:
I made notes on the desired deprecation. LMK if this is good enough for now. I'd like to proceed and at least remove these styles from the shared stylesheet. We'll need to move with deprecation under an experiment in separate pull requests.

Reply:
Sounds good.